### PR TITLE
EZP-29607: Added component Tables to UI Guidelines documentation

### PR DESCRIPTION
**Pull request by inakijv**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [EZP-29607](https://jira.ez.no/browse/EZP-29607)
| Versions      | 2.2.2

Most important aspects:
- Added component Tables to UI Guidelines documentation.
(Needed also to adjust `:disabled` state for buttons within tables).

tables - ez platform developer documentation 1

### CODE REVIEW

**Comment by inakijv** [comment]
Cannot add reviewers (yet) Ping @DominikaK & @sunpietro

File: docs/css/ez-guidelines-buttons.css [hunk 1]
```diff
@@ -15,6 +15,10 @@
     font-size: 1em;
     color: #fff;
 }
+.ez-guidelines .ez-code-example .btn:disabled {
```
Comment (sunpietro) [inline docs/css/ez-guidelines-buttons.css]: What do you think about adding also: `.ez-guidelines .ez-code-example button[disabled]` selector?

File: docs/guidelines/components/tables.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+</div>
+
+!!! Note
+    This table is displayed within `.n-notifications-modal` modal. Styling relies on this class.
+
+
+##<div class="mgt-3 header-line">List table</div>
+<div class="ez-guidelines-tables__list mgb-5">
+[[code_example {html}
+<table class="table ez-table--list">
```
Comment (sunpietro) [inline docs/guidelines/components/tables.md]: It should be `table ez-table ez-table--list`

File: docs/guidelines/components/tables.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    <thead>
+        <tr>
+            <th colspan="2">
```
Comment (sunpietro) [inline docs/guidelines/components/tables.md]: In general, the structure of `ez-table--list` will be updated very soon.

Comment (sunpietro) [reply docs/guidelines/components/tables.md]: Please refer to this PR: https://github.com/ezsystems/ezplatform-admin-ui/pull/614

File: docs/css/ez-guidelines-buttons.css (hunk 1 above)
Comment (inakijv) [reply docs/css/ez-guidelines-buttons.css]: Updated based on review.

File: docs/guidelines/components/tables.md (hunk 2 above)
Comment (inakijv) [reply docs/guidelines/components/tables.md]: Good call @sunpietro. It's true (and I agree with you) that it should be that way, but it's what we currently have:

screen shot 2018-09-14 at 3 39 07 pm

This is also a question for you @DominikaK, Shouldn't we reflect here in the Guidelines the current status of each component? Or the standardized one? I don't have any problem updating the PR, but if this section of the documentation is going to be updated based on releases we should reflect that too. I'd go for this final option. But I need to hear you guys first.

File: docs/guidelines/components/tables.md (hunk 3 above)
Comment (inakijv) [reply docs/guidelines/components/tables.md]: So based on my previous comment, I'd add this new way in the upcoming 2.3 dev-doc branch, but again I need confirmation.

File: docs/guidelines/components/tables.md (hunk 2 above)
Comment (DominikaK) [reply docs/guidelines/components/tables.md]: As a rule we do not name versions in the text, but instead merge changes to proper version branches. So if there is a change that will not apply to 2.2, please add it in a separate PR (to master now, but with a note which versions(s) is should go to).

Comment (inakijv) [reply docs/guidelines/components/tables.md]: Updated to `table ez-table ez-table--list` as requested.

**Comment by inakijv** [comment]
Updated. Ping @DominikaK & @sunpietro

**Pull request merged**